paimon-core/src/main/java/org/apache/paimon/jdbc/JdbcUtils.java

```diff
@@ -0,0 +1,646 @@
+        String[] args =
+                Stream.concat(Stream.of(catalogName, databaseName), properties.stream())
+                        .toArray(String[]::new);
+
+        return execute(connections, JdbcUtils.deletePropertiesStatement(properties), args) > 0;
+    }
+
+    private static String insertPropertiesStatement(int size) {
+        StringBuilder sqlStatement = new StringBuilder(JdbcUtils.INSERT_DATABASE_PROPERTIES_SQL);
+        for (int i = 0; i < size; i++) {
+            if (i != 0) {
+                sqlStatement.append(", ");
+            }
+            sqlStatement.append(JdbcUtils.INSERT_PROPERTIES_VALUES_BASE);
+        }
+        return sqlStatement.toString();
+    }
+
+    private static String deletePropertiesStatement(Set<String> properties) {
+        StringBuilder sqlStatement = new StringBuilder(JdbcUtils.DELETE_DATABASE_PROPERTIES_SQL);
+        String values =
+                String.join(",", Collections.nCopies(properties.size(), String.valueOf('?')));
+        sqlStatement.append("(").append(values).append(")");
+
+        return sqlStatement.toString();
+    }
+
+    private static String updatePropertiesStatement(int size) {
+        StringBuilder sqlStatement =
+                new StringBuilder(
+                        "UPDATE "
+                                + DATABASE_PROPERTIES_TABLE_NAME
+                                + " SET "
+                                + DATABASE_PROPERTY_VALUE
+                                + " = CASE");
+        for (int i = 0; i < size; i += 1) {
+            sqlStatement.append(" WHEN " + DATABASE_PROPERTY_KEY + " = ? THEN ?");
+        }
+        sqlStatement.append(
+                " END WHERE "
+                        + CATALOG_NAME
+                        + " = ? AND "
+                        + DATABASE_NAME
+                        + " = ? AND "
+                        + DATABASE_PROPERTY_KEY
+                        + " IN ");
+
+        String values = String.join(",", Collections.nCopies(size, String.valueOf('?')));
+        sqlStatement.append("(").append(values).append(")");
+        return sqlStatement.toString();
+    }
+
+    public static boolean acquire(JdbcClientPool connections, String lockName, long timeout)
+            throws SQLException, InterruptedException {
+        // Check and clear expire lock
+        int affectedRows = tryClearExpireLock(connections, lockName, timeout);
+        if (affectedRows > 0) {
+            LOG.debug("Successfully cleared " + affectedRows + " lock records");
+        }
+        return connections.run(
+                connection -> {
+                    try (PreparedStatement preparedStatement =
+                            connection.prepareStatement(DISTRIBUTED_LOCK_ACQUIRE_SQL)) {
+                        preparedStatement.setString(1, lockName);
+                        preparedStatement.setTimestamp(
+                                2, new Timestamp(System.currentTimeMillis()));
```

Comment:
So we are using timestamp of clients as `acquired_at`? I think this makes no sense since clients can be distributed on different machines, which have no consistent system time. Since `acquired_at` is the timestamp of acquiring the lock, I think we should use the system time of the metastore.

Reply:
> So we are using timestamp of clients as `acquired_at`? I think this makes no sense since clients can be distributed on different machines, which have no consistent system time. Since `acquired_at` is the timestamp of acquiring the lock, I think we should use the system time of the metastore.

@zhongyujiang There may indeed be the issue you mentioned, but the database used for the meta storage system is different, and the generation system time function used is also different. May require separate compatibility

Reply:
The initial time setting was for metastore, but it was not compatible with all databases, so this version of the modification was made. In fact, specific database specific time functions can be set for each database